Kernel/CMakeLists.txt

```diff
@@ -411,27 +411,6 @@ else()
         ${PREKERNEL_SOURCES}
         ${RPI_SOURCES}
 
-        Arch/aarch64/BootPPMParser.cpp
```

Comment:
Why are we removing these?

Reply:
This was a mistake, I'll put these back.

Reply:
Some of those were duplicated coming from RPI_SOURCES and PREKERNEL_SOURCES